### DISCUSSION TITLE
数据

### DISCUSSION THREAD

**Discussion by swithmn1** [RapidAI/TableStructureRec#91] [Q&A]
数据如何标注的呢

**Comment by Joker1212** [comment]
读光模型直接参考他们项目进行调优就行,wiredV2模型是别人提供的，所以无法直接微调，但是有类似的项目参考https://github.com/AlibabaResearch/AdvancedLiterateMachinery/tree/main/DocumentUnderstanding/LORE-TSR，unet的wiredV2的话参考https://github.com/jiangnanboy/table_structure_recognition?tab=readme-ov-file
